src/style/global.sass

```diff
@@ -31,3 +31,19 @@ ul, ol
     padding-inline: 2rem
     width: 100%
     box-sizing: border-box
+
+.centered-container
+    display: flex
+    width: 100%
+    justify-content: center
+    align-items: center
+
+.super-header
+    color: $color-primary-light
+    font-size: 3rem
+    margin-bottom: 1rem
+    font-weight: bold
+
+@media screen and (min-width: $medium-screen)
+    .super-header
+        font-size: 5rem
```

Comment:
`rem` is a unit that changes based on font-size. In mobile the font-size will be decreased so this isn't needed.

```suggestion
```